android/build_androidtc_arm.sh

```diff
@@ -10,6 +10,22 @@ export NDK_ROOT="${NDK_ROOT:-$HOME/NVPACK/android-ndk-r10e}"
 export ANDROID_TC="${ANDROID_TC:-$HOME/android-tc}"
 export ANDROID_TC_ARM="${ANDROID_TC_ARM:-$ANDROID_TC/arm}"
 
-if [ ! -d $ANDROID_TC_ARM/arm-linux-androideabi ]; then
-    $NDK_ROOT/build/tools/make-standalone-toolchain.sh --platform=android-9 --arch=arm --install-dir=$ANDROID_TC_ARM
+cache_dir=""
+while [[ $# -gt 0 ]]; do
+    key="$1"
+    case $key in
+        --cache_dir)
+        cache_dir="$2"
```

Comment:
You're checking for `cache_dir` but it is ignored in the rest of the script. Is this intentional? This is the same for the other scripts too.

Reply:
I just added this for some possible future cases and to make it the same in all scripts. If you want I can remove this

Reply:
Ok, I just wanted to know if this was intentional. I'm fine with keeping it.